source/Engine.cpp

```diff
@@ -2441,12 +2441,20 @@ void Engine::DoCollection(Flotsam &flotsam)
 	}
 
 	// Unless something went wrong while forming the message, display it.
-	if(!message.empty())
-	{
-		int free = collector->Cargo().Free();
+	if(message.empty()) return;
+
+	int free = collector->Cargo().Free();
+	int total = 0;
+	for(const shared_ptr <Ship> &ship : player.Ships())
+		if(!ship->IsParked() && ship->GetSystem() == player.GetSystem())
```

Comment:
Do we want to include disabled ships?

Reply:
I think I'll leave it as is for now and see if people find that they would prefer disabled ships not be counted or not.